apps/wing/src/commands/test/test.ts

```diff
@@ -521,7 +521,11 @@ async function testSimulator(synthDir: string, options: TestOptions) {
   }
 
   if (clean) {
-    rmSync(synthDir, { recursive: true, force: true });
+    try {
+      rmSync(synthDir, { recursive: true, force: true });
+    } catch (err) {
+      console.warn(`Failed to clean up test directory: ${err}`);
```

Comment:
`warn` is just an alias to `log`, so it may be worth adding extra text and/or using chalk to add some color.

```suggestion
      console.warn(`[WARNING] Failed to clean up test directory: ${err}`);
```